drive/tests/query_tests.rs

```diff
@@ -340,16 +353,19 @@ pub fn add_domains_to_contract(
     }
 }
 
-pub fn setup_dpns_tests_with_batches(count: u32, seed: u64) -> (Drive, Contract, TempDir) {
-    let tmp_dir = TempDir::new().unwrap();
-    let drive: Drive = Drive::open(&tmp_dir, Some(DriveConfig::default_with_batches()))
-        .expect("expected to open Drive successfully");
+pub fn setup_dpns_tests_with_batches(count: u32, seed: u64) -> (Drive, Contract) {
+    let drive = setup_drive(Some(DriveConfig::default_with_batches()));
 
     let db_transaction = drive.grove.start_transaction();
 
+    // Create contracts tree
+    let mut batch = GroveDbOpBatch::new();
+
+    add_init_contracts_tree_operations(&mut batch);
```

Comment:
Why make these changes?

Reply:
Because we used it in tests query as well so we don't need to recalculate app hashes if we update the main initializer with not related to tests stuff.